concepts/promises/about.md

```diff
@@ -1,3 +1,146 @@
 # About
 
-TODO: add information on promises concept
+The [Promise][promise-docs] object represents the eventual completion (or failure) of an asynchronous operation and its resulting value. It allows you to associate handlers with an asynchronous action's eventual success value or failure reason.
```

Comment:
Links can be distracting so they should be used sparingly and only if they add additional value compared to the existing content. E.g. the link to the promise docs above could be placed in the links.json instead (if its not included already). It will then render below the content of this file in the "Read More" section.

Reply:
There is already a link to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Using_promises
Should I also add another link to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise

Reply:
Yes, I think they are different enough so that makes sense.